Source/Meadow.Foundation.Peripherals/ICs.IOExpanders.Ftxxxx/Samples/Ft232h_Sample/Ft232h_Sample.csproj

```diff
@@ -7,12 +7,6 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
-  <ItemGroup>
```

Comment:
I assume this is intentional?

Reply:
Yes.  libmpsse.dll is no longer required for FT232, so it's just removing library cruft